### PR TITLE
Konstantinos / [FE Deriv.com squad] [Markets page] Remove disabled indices from the market offering list

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [x] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/markets/instruments/_european-indices.tsx [hunk 1]
```diff
@@ -1,13 +1,28 @@
 import React from 'react'
 import Symbol from '../components/helper/_symbol'
 import { stocks_european_indices } from '../static/content/_market-symbols'
+import { EU, ROW } from 'components/containers/visibility'
 
+const stocks_european_indices_EU = stocks_european_indices.filter((cfds) => {
```
Comment (mahdiyeh-fs) [inline src/pages/markets/instruments/_european-indices.tsx:6]:
```suggestion
const stocks_european_indices_EU = stocks_european_indices.filter(cfds => !!cfds?.eu === true )
```

Comment (konstantinosG-deriv) [reply src/pages/markets/instruments/_european-indices.tsx]: Ok

File: src/pages/markets/instruments/_european-indices.tsx [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
 const EuropeanIndices = () => {
     return (
         <>
-            {stocks_european_indices.map((symbol, index) => (
-                <Symbol key={index} src={symbol.src} text={symbol.text} />
-            ))}
+            <ROW>
+                <>
+                    {stocks_european_indices.map((symbol, index) => (
+                        <Symbol key={index} src={symbol.src} text={symbol.text} />
```
Comment (michio-rework) [inline src/pages/markets/instruments/_european-indices.tsx:15]: Using map indices as key props is anti-pattern, please check [this ](https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9) out.

File: src/pages/markets/instruments/_european-indices.tsx [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
+                    ))}
+                </>
+            </ROW>
+            <EU>
+                <>
+                    {stocks_european_indices_eu_endpoint.map((symbol, index) => (
+                        <Symbol key={index} src={symbol.src} text={symbol.text} eu={symbol.eu} />
```
Comment (michio-rework) [inline src/pages/markets/instruments/_european-indices.tsx:22]: Using map indices as key props is anti-pattern, please check [this ](https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9) out.

**Review by michio-rework — CHANGES_REQUESTED** [review]

File: src/pages/markets/instruments/_european-indices.tsx (hunk 2 above)
Comment (konstantinosG-deriv) [reply src/pages/markets/instruments/_european-indices.tsx:15]: It was already coded like this, I only wrapped it around ROW and EU components and added a new prop to filter out the unwanted European stock indices.

Comment (prince-deriv) [reply src/pages/markets/instruments/_european-indices.tsx:15]: Maybe you can use `symbol.text` plus the `index` as the value of the `key`

Comment (konstantinosG-deriv) [reply src/pages/markets/instruments/_european-indices.tsx]: Ok will try that

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**